### DISCUSSION TITLE
SPADES

### DISCUSSION THREAD

**Discussion by ilkaybuysal** [ablab/spades#782] [Q&A]
Dear developers,

I'm creating datasets from multiple genomes at 20X coverage and then concatenating these genomes with "cat" to get .R1.fq and .R2.fq files, then using the spades assembler and further testing some other software with the assembled fasta file.

Does the location of reads in .fq files matter for SPADES' algorithm? Would you suggest randomizing the concentration of these multiple .fq files?

Thanks in advance.
Başak

**Comment by asl** [comment]
Hello

No, the location does not matter
